### PR TITLE
Disentangled Representation in Cardiac Image Analysis

### PR DESCRIPTION
**Pull request by nathanpainchaud**
I also updated my review of a follow-up paper to point to this review, instead of the original paper itself.

### CODE REVIEW

File: article/_posts/2020-09-08-DisentangledRepresentationInCardiacImageAnalysis.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+
+# Results
+
+## Semi-supervised segmentation
+![](/article/images/DisentangledRepresentationInCardiacImageAnalysis/table1.jpg)
+
+![](/article/images/DisentangledRepresentationInCardiacImageAnalysis/figure4.jpg)
+
+## Multi-model learning
```
Comment (jodoin) [inline article/_posts/2020-09-08-DisentangledRepresentationInCardiacImageAnalysis.md:94]: modal

**Review by jodoin — APPROVED** [review]

File: article/_posts/2020-08-31-TemporalConsistencyObjectives.md [hunk 2]
```diff
@@ -76,4 +76,4 @@ transformations, the authors split the sequences in two halves:
 temporal coherence, either quantitative or simply qualitative, is provided.
 
 # References
-[^1]: SDNet paper (published in MedIA): [https://arxiv.org/abs/1903.09467](https://arxiv.org/abs/1903.09467)
+[^1]: Review of SDNet: [https://vitalab.github.io/article/2020/09/08/DisentangledRepresentationInCardiacImageAnalysis.html](https://vitalab.github.io/article/2020/09/08/DisentangledRepresentationInCardiacImageAnalysis.html)
```
Comment (AntoineTheb) [inline article/_posts/2020-08-31-TemporalConsistencyObjectives.md:79]: I see you're referencing your own reviews "in the future" hehehe

File: article/_posts/2020-09-08-DisentangledRepresentationInCardiacImageAnalysis.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    authors: "Agisilaos Chartsias, Thomas Joyce, Giorgos Papanastasiou, Michelle Williams, David Newby, Rohan Dharmakumar, Sotirios A. Tsaftaris"
+    title:   "Disentangled Representation Learning in Cardiac Image Analysis"
+    venue:   "Medical Image Analysis (2019)"
+pdf: "https://arxiv.org/pdf/1903.09467.pdf"
+---
+
+
+# Highlights
+- New network architecture (SDNet) that factorizes 2D medical images into spatial anatomical factors and non-spatial
+modality factors;
```
Comment (AntoineTheb) [inline article/_posts/2020-09-08-DisentangledRepresentationInCardiacImageAnalysis.md:16]: "modality factors" is on its own line ?

File: article/_posts/2020-09-08-DisentangledRepresentationInCardiacImageAnalysis.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+- Various experiments to show that the learned representation is well-suited to a variety of image analysis tasks,
+including:
```
Comment (AntoineTheb) [inline article/_posts/2020-09-08-DisentangledRepresentationInCardiacImageAnalysis.md:18]: same as above

**Review by AntoineTheb — APPROVED** [review]
I haven't marked them all but it seems that there's a couple of places where long sentences are separated into their own line in the markdown. However, it doesn't seem to affect the generated markdown view, so I'm approving.

**Comment by nathanpainchaud** [comment]
> I haven't marked them all but it seems that there's a couple of places where long sentences are separated into their own line in the markdown. However, it doesn't seem to affect the generated markdown view, so I'm approving.

Wow! I realize this looks really weird on GitHub's interface. However, all I'm doing is manually wrapping the lines at 120 characters (so that it would display nicely in editors even without line-wrapping turned on). But I agree it looks highly suspicious here... As you said, it doesn't affect how the markdown is processed in the end.

**Pull request merged**